dist/client.js

```diff
@@ -459,7 +459,6 @@
     define([], function() {
       return CrossStorageClient;
     });
-  } else {
-    root.CrossStorageClient = CrossStorageClient;
   }
+  root.CrossStorageClient = CrossStorageClient;
```

Comment:
I appreciate the PR, but I don't think polluting globals is the solution here when in AMD/CommonJS environments. You would encounter this issue with most modules that support those envs, so I don't think the hack should be baked into this library.

As I understand it, you've chosen not to bundle the CrossStorageClient with the electron app, but load it over HTTP instead. Related:
* https://github.com/electron/electron/issues/254

Some options:
* If you're not using node APIs clientside, you could set nodeIntegration to false
* You could install cross-storage in node_modules locally, making it available with `require('cross-storage').CrossStorageClient` (preferred solution)
* If you don't want the client in node_modules, what about downloading the file from the main process on bootup and storing it locally so that it's accessible from your paths/globalPaths
* https://github.com/electron/electron/issues/254#issuecomment-183483641 or https://github.com/electron/electron/issues/254#issuecomment-219204402 or https://github.com/electron/electron/issues/254#issuecomment-208044147
* Maintain a fork of this library that pollutes the global